### PR TITLE
Reorganize menu.

### PR DESCRIPTION
**Pull request by wodenx**

### CODE REVIEW

File: packages/cx-navigation/src/components/Separator/tokens/cxSeparator.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  Spacing: {
+    _: as(
+      'mb-9 pb-9 last:mb-0 last:pb-0',
+      'md:mb-0 md:pb-0 md:px-10 md:first:pl-0 md:last:pr-0',
+      'lg:px-12 lg:first:pl-12',
+    ),
+  },
+});
+console.log('FM', FooterMenu);
```
Comment (gzeni-cit) [inline packages/cx-navigation/src/components/Separator/tokens/cxSeparator.ts:43]: @wodenx ,

There is a console.log here.

**Review by gzeni-cit — APPROVED** [review]

File: packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    Body: cxElements.WithLinkStyles,
+    // Add a class to the icon
+    ExternalIcon: 'text-xl',
+  },
+});
+```
+## Additional Considerations and Gotchas
+
+### Access to component data
+Note the position of hte `Editors` and `Schema` domains which usually provide
```
Comment (hvanyo) [inline packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md]:
```suggestion
Note the position of the `Editors` and `Schema` domains which usually provide
```

File: packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  ```ts
+  const WithBlueBackground = asTokenSpec()({
+    Core: {
+      _: 'bg-blue',
+    },
+    Meta: asToken.meta.term('Color')('Blue'),
+  });
+  ```
+
+## Order ot domains
```
Comment (hvanyo) [inline packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md]:
```suggestion
## Order of domains
```

File: packages/cx-navigation/src/components/Separator/tokens/cxSeparator.ts (hunk 1 above)
Comment (gzeni-cit) [reply packages/cx-navigation/src/components/Separator/tokens/cxSeparator.ts:43]: I'll address it directly in my branch.

File: packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md (hunk 2 above)
Comment (gzeni-cit) [reply packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md]: @hvanyo ,

I'll merge the PR and address it directly in my branch. Thanks!

File: packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md (hunk 3 above)
Comment (gzeni-cit) [reply packages/cx-elements/doc/Design/CanvasxTokens/TokenDomains.md]: @hvanyo ,

I'll merge the PR and address it directly in my branch. Thanks!

**Pull request merged**